Update run-rippled-as-a-validator.md

Tokeiito opened the pull request:
Updated list of official peering hub. Added a note where up to date list could be found.

Tokeiito committed:
Update run-rippled-as-a-validator.md

Updated list of official peering hub. Added a note where up to date list could be found.

maria-robobug commented on docs/infrastructure/configuration/server-modes/run-rippled-as-a-validator.md:
````diff
@@ -207,14 +207,20 @@ This configuration connects your validator to the network using three [public hu
     ```
 
+> **Note:** The above list may evolve over time. To ensure you're using the most current set of public hubs, refer to the official `rippled-example.cfg` maintained by XRPLF:
+> [https://github.com/XRPLF/rippled/blob/develop/cfg/rippled-example.cfg](https://github.com/XRPLF/rippled/blob/develop/cfg/rippled-example.cfg)
````
```suggestion
{% admonition type="info" name="Note" %}
The above list may evolve over time. To ensure you're using the most current set of public hubs, refer to the official [`rippled-example.cfg`](https://github.com/XRPLF/rippled/blob/develop/cfg/rippled-example.cfg) maintained by XRPLF.
{% /admonition %}
```

maria-robobug commented on docs/infrastructure/configuration/server-modes/run-rippled-as-a-validator.md:
```diff
@@ -207,14 +207,20 @@ This configuration connects your validator to the network using three [public hu
 _**To connect your validator to the network using public hubs:**_
 
-1. In your validator's `rippled.cfg` file, include the following `[ips_fixed]` stanza. The two values, `r.ripple.com 51235`, and `sahyadri.isrdc.in 51235`, are default public hubs. This stanza tells `rippled` to always attempt to maintain peer connections with these public hubs.
+1. In your validator's `rippled.cfg` file, include the following `[ips_fixed]` stanza. The two values, `r.ripple.com 51235`, and `sahyadri.isrdc.in 51235`, are default public hubs. This stanza tells `rippled` to always attempt to maintain peer connections with these public hubs. 
```
```suggestion
1. In your validator's `rippled.cfg` file, include the following `[ips_fixed]` stanza. The two values, `r.ripple.com 51235` and `sahyadri.isrdc.in 51235`, are default public hubs. This stanza tells `rippled` to always attempt to maintain peer connections with these public hubs.
```

Tokeiito replied:
I've simplified that sentece by removing addresses.

maria-robobug reviewed (dismissed):
Minor suggestions, but LGTM

Tokeiito committed:
Update docs/infrastructure/configuration/server-modes/run-rippled-as-a-validator.md

Changing style of note to use admonition as sugested

Co-authored-by: Maria Shodunke <maria-robobug@users.noreply.github.com>

Tokeiito committed:
Update run-rippled-as-a-validator.md

Siplified description of [ips_fixed] stanza. Removed unnecessary listing of domains as that was duplicated example bellow.
